RevenueCatUI/Templates/V2/Components/Stack/StackComponentView.swift

```diff
@@ -112,13 +115,52 @@ struct StackComponentView: View {
                uiConfigProvider: self.viewModel.uiConfigProvider)
         .apply(badge: style.badge, border: style.border, shadow: style.shadow, shape: style.shape)
         .padding(style.margin)
+        .scrollableIfEnabled(
+            style.dimension,
+            enabled: style.scrollable ?? self.isScrollableByDefault
+        )
+    }
+
+}
+
+private extension Axis {
+
+    var scrollViewAxis: Axis.Set {
+        switch self {
+        case .horizontal: return .horizontal
+        case .vertical: return .vertical
```

Comment:
Just to make sure, we won’t allow scrolling in the opposite direction than the axis of the stack then? I think that can make sense for now (in the android PR I was assuming a direction would be given by the backend to make it more flexible… but we might not need that flexibility)

Reply:
@tonidero Correct correct! That might make it a bit more confusing to use 😅